test/test_jit.py

```diff
@@ -1712,7 +1720,7 @@ def constant_prop():
             return b
 
         self.run_pass('constant_propagation', constant_prop.graph)
-        self.assertExpected(canonical(constant_prop.graph))
```

Comment:
what is this test really checking here? I think aten::randn will always be there without constant prop or not.

Reply:
This was testing that randn was not run. DCE gets run after constant prop so if it was run it would no longer be in the graph